solidity/contracts/system/TBTCSystem.sol

```diff
@@ -566,6 +562,33 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         );
     }
 
+    /// @notice Refreshes the minimum bondable value required from the operator
+    /// to join the sortition pool for tBTC. The minimum bondable value is
+    /// equal to the current minimum lot size collateralized 150% multiplied by
+    /// the current BTC price.
+    /// @dev It is recommended to call this function on tBTC initialization and
+    /// after minimum lot size update.
+    function refreshMinimumBondableValue() public {
+        uint256 bondRequirementSatoshis = getMinimumLotSizeSatoshi().mul(
+            initialCollateralizedPercent
+        ).div(100);
+        uint256 bondRequirementWei = _fetchBitcoinPrice().mul(
+            bondRequirementSatoshis
+        );
+        keepFactorySelection.setMinimumBondableValue(bondRequirementWei);
+    }
+
+    /// @notice Returns the minimum lot size in satoshi.
+    function getMinimumLotSizeSatoshi() public view returns (uint256) {
+        uint256 minimum = lotSizesSatoshis[0];
```

Comment:
It feels like we'd do better to ensure the lot size array is in order when we set it, rather than trying to sort it later?

Reply:
Yeah, I wasn't sure if I can do that. The mandatory lot size looks to me like something we might always pass as the very first value just for convenience but given that it's not a function we call every day, expecting lot sizes to come sorter makes sense. If you agree with this change, I am happy to update the code 🙌

Reply:
Done in https://github.com/keep-network/tbtc/pull/714/commits/5ec1459464c293954f86bd5edea6f35d9f92bf99.